### ISSUE TITLE
NTR: Dihydrouracil high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8814]
New term request Dihydrouracil high in urine (segal_060511203313):   UMLS: 1847704

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000118
- label: Elevated urinary dihydrouracil level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: PMID:9323563
- def: The amount of dihydrouracil in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8814
- synonyms: Dihydrouracil high in urine(NONE)
